### ISSUE TITLE
some instances don't have mask

### ISSUE THREAD

**Issue by Joboing** [youngwanLEE/CenterMask#18]
I use centermask_demo.py to test my model，i find some instance result don't have mask.

**Comment by youngwanLEE** [comment]
@Joboing,

Why don't you adjust the [confident_threshold ](https://github.com/youngwanLEE/CenterMask/blob/master/demo/centermask_demo.py#L37)?

**Comment by Joboing** [comment]
> @Joboing,
> 
> Why don't you adjust the [confident_threshold ](https://github.com/youngwanLEE/CenterMask/blob/master/demo/centermask_demo.py#L37)?

Thanks for your reply. Now my problem is that the instance has a box without a mask. I also adjust the confident_threshold ,it doesn't work. The FCOS BOX Head is fine，Whether the parameters of mask head need to be adjusted?

**Comment by youngwanLEE** [comment]
@Joboing 
It is strange that boxes are detected normally but the masks are not predictable.

Did you show me your config file?

Did you set the [MASK_ON](https://github.com/youngwanLEE/CenterMask/blob/master/demo/predictor.py#L230-L231) to True?

**Comment by Joboing** [comment]
> @Joboing
> It is strange that boxes are detected normally but the masks are not predictable.
> 
> Did you show me your config file?
> 
> Did you set the [MASK_ON](https://github.com/youngwanLEE/CenterMask/blob/master/demo/predictor.py#L230-L231) to True?
The follow is my config file
MODEL:
  META_ARCHITECTURE: "GeneralizedRCNN"
  WEIGHT: "/home/centermask/vovnet39_ese_state_dict.pth"
  BACKBONE:
    CONV_BODY: "V-39-eSE-FPN-RETINANET"
    FREEZE_CONV_BODY_AT: 0
  VOVNET:
    BACKBONE_OUT_CHANNELS: 256
  RPN_ONLY: True
  FCOS_ON: True
  FCOS_MASK: True
  RETINANET:
    USE_C5: False # FCOS uses P5 instead of C5
    NUM_CLASSES: 6
  MASK_ON: True
  MASKIOU_ON: True
  FCOS:
    CENTER_SAMPLE: True
    POS_RADIUS: 1.5
    LOC_LOSS_TYPE: "giou"
    INFERENCE_TH: 0.03
    NUM_CLASSES: 6
  ROI_HEADS:
    USE_FPN: True
  ROI_MASK_HEAD:
    POOLER_SCALES: (0.125, 0.0625, 0.03125) # 1/8, 1/16, 1/32
    FEATURE_EXTRACTOR: "MaskRCNNFPNSpatialAttentionFeatureExtractor"
    LEVEL_MAP_FUNCTION: "CenterMaskLevelMapFunc"
    PREDICTOR: "MaskRCNNC4Predictor"
    POOLER_RESOLUTION: 14
    POOLER_SAMPLING_RATIO: 2
    RESOLUTION: 28
    SHARE_BOX_FEATURE_EXTRACTOR: False
DATASETS:
  TRAIN: ("coco_2014_train",)
  TEST: ("coco_2014_val",)
INPUT:
MIN_SIZE_RANGE_TRAIN: (800,800)
  MAX_SIZE_TRAIN: 1333
  MIN_SIZE_TEST: 800
  MAX_SIZE_TEST: 1333
DATALOADER:
  SIZE_DIVISIBILITY: 32
SOLVER:
  BASE_LR: 0.002
  WEIGHT_DECAY: 0.0001
  STEPS: (120000, 140000)
  MAX_ITER: 160000
  IMS_PER_BATCH: 2
  CHECKPOINT_PERIOD: 20000
  WARMUP_METHOD: "constant"
TEST:
  IMS_PER_BATCH: 2
OUTPUT_DIR : .

**Comment by Joboing** [comment]
> > @Joboing
> > It is strange that boxes are detected normally but the masks are not predictable.
> > Did you show me your config file?
> > Did you set the [MASK_ON](https://github.com/youngwanLEE/CenterMask/blob/master/demo/predictor.py#L230-L231) to True?
> > The follow is my config file
> > MODEL:
> > META_ARCHITECTURE: "GeneralizedRCNN"
> > WEIGHT: "/home/centermask/vovnet39_ese_state_dict.pth"
> > BACKBONE:
> > CONV_BODY: "V-39-eSE-FPN-RETINANET"
> > FREEZE_CONV_BODY_AT: 0
> > VOVNET:
> > BACKBONE_OUT_CHANNELS: 256
> > RPN_ONLY: True
> > FCOS_ON: True
> > FCOS_MASK: True
> > RETINANET:
> > USE_C5: False # FCOS uses P5 instead of C5
> > NUM_CLASSES: 6
> > MASK_ON: True
> > MASKIOU_ON: True
> > FCOS:
> > CENTER_SAMPLE: True
> > POS_RADIUS: 1.5
> > LOC_LOSS_TYPE: "giou"
> > INFERENCE_TH: 0.03
> > NUM_CLASSES: 6
> > ROI_HEADS:
> > USE_FPN: True
> > ROI_MASK_HEAD:
> > POOLER_SCALES: (0.125, 0.0625, 0.03125) # 1/8, 1/16, 1/32
> > FEATURE_EXTRACTOR: "MaskRCNNFPNSpatialAttentionFeatureExtractor"
> > LEVEL_MAP_FUNCTION: "CenterMaskLevelMapFunc"
> > PREDICTOR: "MaskRCNNC4Predictor"
> > POOLER_RESOLUTION: 14
> > POOLER_SAMPLING_RATIO: 2
> > RESOLUTION: 28
> > SHARE_BOX_FEATURE_EXTRACTOR: False
> > DATASETS:
> > TRAIN: ("coco_2014_train",)
> > TEST: ("coco_2014_val",)
> > INPUT:
> > MIN_SIZE_RANGE_TRAIN: (800,800)
> > MAX_SIZE_TRAIN: 1333
> > MIN_SIZE_TEST: 800
> > MAX_SIZE_TEST: 1333
> > DATALOADER:
> > SIZE_DIVISIBILITY: 32
> > SOLVER:
> > BASE_LR: 0.002
> > WEIGHT_DECAY: 0.0001
> > STEPS: (120000, 140000)
> > MAX_ITER: 160000
> > IMS_PER_BATCH: 2
> > CHECKPOINT_PERIOD: 20000
> > WARMUP_METHOD: "constant"
> > TEST:
> > IMS_PER_BATCH: 2
> > OUTPUT_DIR : .

**Comment by Joboing** [comment]
> > @Joboing
> > It is strange that boxes are detected normally but the masks are not predictable.
> > Did you show me your config file?
> > Did you set the [MASK_ON](https://github.com/youngwanLEE/CenterMask/blob/master/demo/predictor.py#L230-L231) to True?
> > The follow is my config file
> > MODEL:
> > META_ARCHITECTURE: "GeneralizedRCNN"
> > WEIGHT: "/home/centermask/vovnet39_ese_state_dict.pth"
> > BACKBONE:
> > CONV_BODY: "V-39-eSE-FPN-RETINANET"
> > FREEZE_CONV_BODY_AT: 0
> > VOVNET:
> > BACKBONE_OUT_CHANNELS: 256
> > RPN_ONLY: True
> > FCOS_ON: True
> > FCOS_MASK: True
> > RETINANET:
> > USE_C5: False # FCOS uses P5 instead of C5
> > NUM_CLASSES: 6
> > MASK_ON: True
> > MASKIOU_ON: True
> > FCOS:
> > CENTER_SAMPLE: True
> > POS_RADIUS: 1.5
> > LOC_LOSS_TYPE: "giou"
> > INFERENCE_TH: 0.03
> > NUM_CLASSES: 6
> > ROI_HEADS:
> > USE_FPN: True
> > ROI_MASK_HEAD:
> > POOLER_SCALES: (0.125, 0.0625, 0.03125) # 1/8, 1/16, 1/32
> > FEATURE_EXTRACTOR: "MaskRCNNFPNSpatialAttentionFeatureExtractor"
> > LEVEL_MAP_FUNCTION: "CenterMaskLevelMapFunc"
> > PREDICTOR: "MaskRCNNC4Predictor"
> > POOLER_RESOLUTION: 14
> > POOLER_SAMPLING_RATIO: 2
> > RESOLUTION: 28
> > SHARE_BOX_FEATURE_EXTRACTOR: False
> > DATASETS:
> > TRAIN: ("coco_2014_train",)
> > TEST: ("coco_2014_val",)
> > INPUT:
> > MIN_SIZE_RANGE_TRAIN: (800,800)
> > MAX_SIZE_TRAIN: 1333
> > MIN_SIZE_TEST: 800
> > MAX_SIZE_TEST: 1333
> > DATALOADER:
> > SIZE_DIVISIBILITY: 32
> > SOLVER:
> > BASE_LR: 0.002
> > WEIGHT_DECAY: 0.0001
> > STEPS: (120000, 140000)
> > MAX_ITER: 160000
> > IMS_PER_BATCH: 2
> > CHECKPOINT_PERIOD: 20000
> > WARMUP_METHOD: "constant"
> > TEST:
> > IMS_PER_BATCH: 2
> > OUTPUT_DIR : .

111
